source/blog/2018-07-13-the-ember-times-issue-55.md

```diff
@@ -0,0 +1,136 @@
+---
+title: The Ember Times - Issue No. 55
+author: Kenneth Larsen, Amy Lam, Miguel Braga Gomes, Ryan Mark, Sivakumar Kailasam, Jessica Jordan, Alon Bukai
+tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2018
+alias : "blog/2018/07/13-the-ember-times-issue-55.html"
+responsive: true
+---
+
+nuqneH Emberistas! 🐹
+
+Read either on the [Ember blog](https://www.emberjs.com/blog/2018/07/13/the-ember-times-issue-55.html) or in our [e-mail newsletter](https://the-emberjs-times.ongoodbits.com/2018/07/13/the-ember-times-issue-55) what has been going on in Emberland this week.
+
+This week you can learn about **updating** your ember-cli app 🖳. Learn from firsthand experience how to become an **addon maintainer** 💪. Get some tips on writing your own **RFCs** ✨. Check out the new **SEO tactics** of Ember 🔎 and last but not least learn about the potential move of Ember to a new **chat platform** 💬. Go ahead and enjoy!
+
+---
+
+## [Don't Worry, Ember CLI Got You Covered](https://github.com/ember-cli/ember-cli-update) 🖳
```

Comment:
It seems the emoji got lost here in the header title - can you check?

Reply:
It is supposed to be a computer. UP works as well though

Reply:
It seems the emoji is still not [displayed correctly](https://ember-website-staging-pr-3452.herokuapp.com/blog/2018/07/13/the-ember-times-issue-55.html#toc_a-href-https-github-com-ember-cli-ember-cli-update-don-t-worry-ember-cli-got-you-covered-a) - I sent an update for it here: https://github.com/emberjs/website/pull/3457 Can you take a :eyes: ?